src/api/user/user.controller.ts

```diff
@@ -21,7 +34,12 @@
   @Roles(ROLE_TYPE.SUPER_ADMIN)
   @ApiOperation({ summary: 'Create a new user' })
   @ApiResponse({ status: 201, description: 'User created successfully.' })
-  async createUser(@Body() createUserDto: CreateUserDto) {
+  async CreateUser(@Body() createUserDto: CreateUserDto) {
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/naming-convention](https://typescript-eslint.io/rules/naming-convention)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class Method name `CreateUser` must match one of the following formats: camelCase
<!-- __reviewdog__:ChAzMzcxZTk1ZGIwNjJkM2Q3EgZlc2xpbnQ= -->